.github/workflows/docker.yml

```diff
@@ -232,12 +247,14 @@ jobs:
           GITEA_DEPLOY_IMAGE_TAG: ':${{ steps.sha.outputs.gitea_sha_short }}'
         with:
           start: docker-compose -f ../docker-compose.yml -f ../docker-compose.deploy.yml up
-          command: yarn e2e --record --browser chrome
-          working-directory: frontend
           # wait for 2 minutes for the server to respond
           wait-on: 'http://kitspace.test:3000'
           # This value is in seconds
-          wait-on-timeout: 120
+          wait-on-timeout: 120         
+          working-directory: frontend
+          record: true
+          browser: ${{ matrix.browsers }}
```

Comment:
Why did you change this to use `command` before? I was non-command before then you changed it to be command and now back again. Just trying to understand...

Reply:
If the `command` key is present, the run options e.g., `record`, and `browser` are ignored.
In 6356f81088aad69abb363e3adf685d4f700fb1c4, I thought there was a problem with the run options, turned out I used `matrix.browser` instead of `matrix.browsers`.